node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -537,6 +550,7 @@ instance Migrate WalletStorage where
         , _wsTxHistory       = _v0_wsTxHistory
         , _wsHistoryCache    = HM.map txHistoryListToMap _v0_wsHistoryCache
         , _wsUtxo            = _v0_wsUtxo
+        , _wsBalances        = utxoToAddressCoinMap _v0_wsUtxo
```

Comment:
I don't think that you should alter already existing migration - this wouldn't work, because customers already applied first migration. You should remake this PR so that it's based on current tip of `cardano-sl-1.0-exchanges` and adds second migration on top of first one.